src/docs/arch-log-file.dox

```diff
@@ -1,7 +1,7 @@
 /*! @arch_page arch-log-file Log File Format
 
 This page assumes familiarity with the logging subsystem. Please see
-@ref arch-logging for the logging subsystem architecture description.
+the @ref arch-logging "Logging" architecture for a description of that subsystem.
```

Comment:
"the Logging architecture" feels awkward, maybe just Please see "Logging"

Reply:
Done